lesson_07/homework_07.py

```diff
@@ -68,4 +122,56 @@ def find_substring(str1, str2):
 """  Оберіть будь-які 4 таски з попередніх домашніх робіт та
 перетворіть їх у 4 функції, що отримують значення та повертають результат.
 Обоязково документуйте функції та дайте зрозумілі імена змінним.
-"""
\ No newline at end of file
+"""
+
+
+def duplicate_list_checker(ch_list: list) -> bool:
+    """Checks the list for duplicates"""
+    ch_set = set(ch_list)
+    if len(ch_list) > len(ch_set):
+        return True
+    return False
```

Comment:
```suggestion
def is_duplicate_in_list(ch_list: list) -> bool:
    """Checks the list for duplicates"""
    ch_set = set(ch_list)
    return len(ch_list) > len(ch_set)
```
ф-ції що повертають булеів значення як правло називають з is 
ну  також if там не те щоб прям нада